src/app/home/home.component.ts

```diff
@@ -1,62 +1,44 @@
 import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { MapViewInfoDto } from './map-view/map-view.dto';
-import { DashboardViewInfoDto } from './dashboard-view/dashboard-view.dto';
-
 import { Subscription } from 'rxjs';
-import { ViewInfoDto } from './view.dto';
+
+import { ViewDto } from './view.dto';
 
 @Component({
     selector: 'sh-home',
     templateUrl: './home.template.html',
     styleUrls: [ './home.style.scss' ]
 })
 export class HomeComponent {
-    public listMapViews: Array<MapViewInfoDto> = [];
-    public listDashboardViews: Array<DashboardViewInfoDto> = [];
-    public currentView: MapViewInfoDto | DashboardViewInfoDto;
-    private defaultResolver: Subscription;
+    public viewList: Array<ViewDto> = [];
+    public currentView: ViewDto;
+    public currentSubview: string;
+
+    private activeRouteDataSubscription: Subscription;
 
     constructor(private activeRoute: ActivatedRoute) {};
 
     ngOnInit() {
-        this.defaultResolver = this.activeRoute.data.subscribe(({viewList}) => {
-            viewList.map(view => this.setCrutch(view));
-
-            this.listMapViews = viewList
-                .filter(view => !!view.mapView)
-                .map(view => view.mapView);
-
-            this.listDashboardViews = viewList
-                .filter(view => !!view.dashboard)
-                .map(view => view.dashboard);
-
-            this.currentView = this.listMapViews.find(view => view.default) || this.listMapViews[0];
+        this.activeRouteDataSubscription = this.activeRoute.data.subscribe(({viewList}) => {
+            this.viewList = viewList;
+            this.setCurrentView(viewList[0]);
```

Comment:
We discussed that on home page we should display view which the default flag equals true.